doc/classes/ScriptEditor.xml

```diff
@@ -34,6 +34,13 @@
 				Returns an array with all [Script] objects which are currently open in editor.
 			</description>
 		</method>
+		<method name="goto_help">
+			<return type="void" />
+			<param index="0" name="topic" type="String" />
+			<description>
+				Opens help for the given topic.
```

Comment:
It would be good to add details on what the topic is meant to be, with some examples.

Reply:
Hi @akien-mga, that's reasonable.  Just one question, I was considering appending this:
```
For example: [code]class_name:<class_name>[/code] or [code]class_method:<class_name>:<method_name>[/code].
```
My concern in this context are the `<` and `>` and how they should be represented.  I also tried to find some other examples in the repository and it wasn't immediately obvious of any I could use as a basis for how best to represent this type of example where there is expected tokens combined with variable user-supplied data.  If there is, could you point me to a reference?

Reply:
So the closest I found was in `EditorCommandPalette` that uses:
```
[code]"example/command1"[/code] then [code]example[/code] will be the section name.
```
If that's acceptable here in this context, I could use `"class"` and `"method"` to represent the user-supplied values to distinguish it from the expected static values that the help system expects.  wdyt?

Reply:
Updated, let me know if it needs any further adjustments.

Reply:
That looks ok, but there seems to be more types of topics supported by this method, see:

https://github.com/godotengine/godot/blob/7e4c150573d6af7072f2b55ae76dc7b723f21f66/editor/editor_help.cpp#L2246-L2306

So it should likely be a list or supported tags with their expected arguments.

And then a few real life examples like `class_method:Node:add_child`.

Would also be good to figure out what's the format used for GDScript class docs so that this could be shown as an example too (maybe it's just the registered `class_name`, I didn't check).

Reply:
So this is what I have come up with @AThousandShips / @akien-mga.  It's a bit wordy, so I'm certainly open to suggestions, but I believe it conveys the details you requested.

![image](https://github.com/godotengine/godot/assets/3255568/0cc61f37-14f2-4d2a-876d-c02d88dff637)

Reply:
That looks really good!